src/pages/shared/window-buttons/window-buttons.html

```diff
@@ -1,124 +1,118 @@
 <!DOCTYPE html>
 <html>
+<head>
     <title>YouTube Music</title>
-    <head>
-        <meta charset="UTF-8" />
-        <link
-            href="../../../../src/assets/css/material_icons.css"
-            rel="stylesheet"
-        />
-        <link
-            type="text/css"
-            rel="stylesheet"
-            href="../../../../src/assets/css/styles.css"
-        />
-        <link
-            type="text/css"
-            rel="stylesheet"
-            href="../../../../src/assets/css/materialize.min.css"
-            media="screen,projection"
-        />
-        <link type="text/css" rel="stylesheet" href="./window-buttons.css" />
-
-        <link rel="icon" href="../../../../src/assets/img/favicon.png" />
-    </head>
+    <meta charset="utf-8" />
+    <link
+        type="text/css"
+        href="../../../../src/assets/css/material_icons.css"
+        rel="stylesheet"
+    />
+    <link
+        type="text/css"
+        rel="stylesheet"
+        href="../../../../src/assets/css/styles.css"
+    />
+    <link
+        type="text/css"
+        rel="stylesheet"
+        href="../../../../src/assets/css/materialize.min.css"
+        media="screen,projection"
+    />
+    <link type="text/css" rel="stylesheet" href="./window-buttons.css" />
+    <link rel="icon" type="image/png" href="../../../../src/assets/img/favicon.png" />
+</head>
+<body>
+    <div id="nice-titlebar" class="bar drag">
+        <div id="mac" class="hide no-drag">
+            <span id="window-title" class="hide"></span>
+        </div>
 
-    <body>
-        <div id="nice-titlebar" class="bar drag">
-            <div id="mac" class="hide no-drag">
-                <span id="window-title" class="hide"></span>
-                <!-- <div class="col s6 center">
+        <div id="win" class="hide">
+            <div class="row">
+                <div class="col s9 left-align">
                     <div class="title">
```

Comment:
these changes were you able to test on windows, mac and linux? I'm afraid they won't work properly on any of them, I can test it on windows.

Reply:
Hi @adlerluiz, I just deleted the [comment](https://github.com/ytmdesktop/ytmdesktop/pull/757/commits/5d7c662328123437293eb62befae272926695dde). :)

But you can check the changes.

Reply:
sorry, i didn't refer to the comment but the div structure because the window is different in windows and in linux... but i'll test here in windows and linux =)